spec/observables/ScalarObservable-spec.ts

```diff
@@ -18,11 +19,14 @@ describe('ScalarObservable', () => {
   });
 
   it('should not schedule further if subscriber unsubscribed', () => {
-    const s = new ScalarObservable(1, rxTestScheduler);
+    const schedulerMock = sinon.stub(rxTestScheduler);
+    const s = new ScalarObservable(1, schedulerMock as any);
     const subscriber = new Rx.Subscriber();
     s.subscribe(subscriber);
-    subscriber.closed = true;
+    subscriber.unsubscribe();
     rxTestScheduler.flush();
+
+    expect((schedulerMock as any).schedule).calledOnce;
```

Comment:
while looking codes I noticed this test case actually did not assert anything, modified to have assertion but not sure if this is intended one - will update accordingly as suggested.